### PR TITLE
WIP: Automated e2e tests initial infrastructure setup

### PR DESCRIPTION
**Pull request by Sergiu-B**
# Proposed Changes
* Implemented tests for the homepage based on the existing feature file;
* Configured chromedriver;
* Added script for running the implemented tests;
* Updated README file with the necessary info for running the tests;

# Notes
I wanted to avoid a huge PR, so I thought about implementing the other tests plus support for multiple browsers and mobile in different PRs.

### CODE REVIEW

File: tests_root/wdio/configs/wdio.conf.js [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+  // from which `wdio` was called. Notice that, if you are calling `wdio` from an
+  // NPM script (see https://docs.npmjs.com/cli/run-script) then the current working
+  // directory is where your package.json resides, so `wdio` will be called from there.
+  //
+  specs: [
+    './features/**/*.feature'
+  ],
+  // Patterns to exclude.
+  exclude: [
+    './features/check_account.feature',
```
Comment (Sergiu-B) [inline tests_root/wdio/configs/wdio.conf.js]: These tests are not implemented yet.

File: tests_root/wdio/configs/wdio.conf.js [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+  bail: 0,
+  //
+  // Saves a screenshot to a given path if a command fails.
+  screenshotPath: './errorShots/',
+  //
+  // Set a base URL in order to shorten url command calls. If your `url` parameter starts
+  // with `/`, the base url gets prepended, not including the path portion of your baseUrl.
+  // If your `url` parameter starts without a scheme or `/` (like `some/path`), the base url
+  // gets prepended directly.
+  baseUrl: process.env.baseUrl || 'http://localhost:8080/',
```
Comment (Sergiu-B) [inline tests_root/wdio/configs/wdio.conf.js]: At one point I assume we'll be providing different URLs per environment.

File: tests_root/wdio/features/home.feature [hunk 3]
```diff
@@ -0,0 +1,12 @@
+@Home_Screen
+Feature: Home Screen of Beep Web App
+  As a User
+  I want to navigate to the appropriate screen
+  So that I can check if my account name or password have been compromised
+
+  @S1 @manual
+  Scenario: Verification on fields on screen
```
Comment (Sergiu-B) [inline tests_root/wdio/features/home.feature]: Removed data from the feature file. This should not contain any implementation details.

File: tests_root/wdio/features/home.feature [hunk 4]
```diff
@@ -0,0 +1,12 @@
+@Home_Screen
+Feature: Home Screen of Beep Web App
+  As a User
+  I want to navigate to the appropriate screen
+  So that I can check if my account name or password have been compromised
+
+  @S1 @manual
```
Comment (popescunsergiu) [inline tests_root/wdio/features/home.feature]: If this test is automated, please remove the @manual tag

**Comment by grgur** [comment]
This looks great @Sergiu-B, thank you for contributing!
Can you also add this as a step in CircleCI?

File: tests_root/wdio/features/home.feature (hunk 4 above)
Comment (Sergiu-B) [reply tests_root/wdio/features/home.feature]: Oops! Good catch! 
Addressed in commit aa2beb4.

**Comment by Sergiu-B** [comment]
>Can you also add this as a step in CircleCI?
@grgur Sure thing! As long as I have permissions :)

**Pull request merged**